src/um/shobjidl.rs

```diff
@@ -382,3 +385,31 @@ interface IFileDialog2(IFileDialog2Vtbl): IFileDialog(IFileDialogVtbl) {
         psi: IShellItem,
     ) -> HRESULT,
 });
+
+RIDL!(
```

Comment:
Item level macros should use `{}` and there should be no newline between `RIDL!{` and `#[uuid]`.